packages/website/pages/files.js

```diff
@@ -96,6 +101,118 @@ export default function Files({ user }) {
 
   const hasZeroNfts = nfts.length === 0 && befores.length === 1
 
+  /**
+   * @param {any} nft
+   */
+  const TableItem = ({ nft }) => {
```

Comment:
TableItem seems to be large and decently complex, we should probably move this to it's own file/seperate component to reduce the noise in this one.

Reply:
Refactor request, can be solved another time.